backend/db_migrations/1584981130161_add-players-have-same-game-id-as-match.sql

```diff
@@ -0,0 +1,23 @@
+CREATE FUNCTION MATCHES_AFTER_INSERT_TRIGGER_CREATE()
+RETURNS trigger
+LANGUAGE PLPGSQL AS
+$$BEGIN
+  IF EXISTS (
+    SELECT * FROM "Matches" m, "Players" p
```

Comment:
can't you use inner join between matches and players here? 🤔

Reply:
it is an inner join :relaxed: https://www.postgresql.org/docs/8.3/tutorial-join.html

Reply:
yes, I know. I'm suggesting to use the dedicated syntax for it, so the intent is even more clear

Reply:
no join anymore :) https://github.com/salsita/foosball-rating/pull/129/commits/2d493ce517f94386c34244ee51f814d41ee6896b